### ISSUE TITLE
CIS Scan - Rancher should prevent running CIS Scans in clusters with k8s version lower than 1.15

### ISSUE THREAD

**Issue by izaac** [rancher/rancher#25996]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**

- Cluster after upgrading from 2.2.9 or 2.3.x with k8s lower than 1.15
- `runSecurityScan` from API or Run CIS Scan from UI. 

**Result:**

The CIS Scan gets executed even if the k8s version is lower than 1.15

**Expectation:**

Prevent the user from running scans in clusters with k8s versions for CIS 1.4 Profile

**Environment information**
- Rancher version: `2.4 master-head (03/12/2020)` commit id: d47b0ec4c
- Installation option (single install/HA): single

**Comment by izaac** [comment]
Rancher version: `v2.4 master-head (03/20/2020)` commit id: 3e6e9ddee Upgraded from Rancher v2.2.10.

I tried this with three clusters versions upgraded from v2.2.10:

- v1.13.12
- v1.14.8
- v1.15.5

Versions lower than 1.15.x shown the following message when tried to run the CIS Scan:

![Screen Shot 2020-03-20 at 5 12 42 PM](https://user-images.githubusercontent.com/27659/77214930-a6a72300-6ace-11ea-8313-88d10173ced5.png)

But I was able to trigger a CIS Scan from the API, but it just got stuck in the `Creating` state:

![Screen Shot 2020-03-20 at 5 19 39 PM](https://user-images.githubusercontent.com/27659/77214983-ff76bb80-6ace-11ea-80f2-722a0405e3c4.png)


The difference we noticed when doing from UI and API was the UI is sending the Action POST as:

```json
{
	"failuresOnly": false,
	"skip": null,
	"profile": "permissive",
	"overrideBenchmarkVersion": "rke-cis-1.4"
}
```

And the API action (with defaults) as:

```json
{

    "debugMaster": false,
    "debugWorker": false,
    "overrideBenchmarkVersion": null,
    "overrideSkip": [ ],
    "profile": "permissive"
}
```

I was able to run scans from the UI and API for k8s version 1.15, 1.16 and 1.17. Clusters with k8s 1.16 and 1.17 were created after upgrade.

Reopening because I was able to trigger scans from the API for clusters with k8s versions 1.13 and 1.14

**Comment by izaac** [comment]
Rancher version: v2.4 master-head (03/21/2020) commit id: 546b039e4 Upgraded from Rancher v2.2.10.

I tried this with three clusters versions upgraded from v2.2.10:

- v1.13.12
- v1.14.8
- v1.15.5

Versions lower than 1.15.x shown the following message when tried to run the CIS Scan (`Global`, `Cluster Dashboard` and `Tools -> CIS Scans` pages:

<img width="236" alt="Screen Shot 2020-03-21 at 7 15 13 PM" src="https://user-images.githubusercontent.com/27659/77240873-51d7db00-6ba8-11ea-974c-7afa08794987.png">


Now triggering the `runSecurityScan` action from the API with default values return (expected): 

```json
{
"baseType": "error",
"code": "MethodNotAllow",
"message": "minimum k8s version 1.15 needed for running cis scan",
"status": 405,
"type": "error"
}
```

Same if I use `overrideBenchmarkVersion": "rke-cis-1.4"` in `View API` or different value for the `profile` key (hardened or permissive).


I was able to run scans from the `UI` and `View API` for k8s version 1.15, 1.16 and 1.17. Clusters with k8s 1.16 and 1.17 were created after upgrade.

Closing as this is now meeting the requirement.
